scripts/make_guide_images.ts

```diff
@@ -0,0 +1,48 @@
+import { GUIDES } from "../src/routes/guides/urls.ts"
+import CURRENT_IMAGES from "../src/routes/guides/images.json"
+
+import * as fs from "fs"
+
+// path -> name
+const guidesMap: { [key: string]: string } = {}
+Object.entries(GUIDES).forEach(([name, guide]) => {
+  guidesMap[guide.href] = name
+})
+
+const imgMap: { [key: string]: string } = {}
+Object.entries(CURRENT_IMAGES).forEach(([guideName, images]) => {
+  Object.entries(images).forEach(([imgName, image]) => {
+    imgMap[image.src] = imgName
+  })
+})
+
+const GUIDES_IMG_DIR = "static/guides"
+
+const images = {}
```

Comment:
At least my PyCharm is saying this `images` is unused and I think I agree with it.